plugins/imtcp/imtcp.c

```diff
@@ -549,6 +556,7 @@ CODESTARTendCnfLoad
 		pModConf->bSuppOctetFram = cs.bSuppOctetFram;
 		pModConf->iAddtlFrameDelim = cs.iAddtlFrameDelim;
 		pModConf->bDisableLFDelim = cs.bDisableLFDelim;
+		pModConf->discardTruncatedMsg = cs.discardTruncatedMsg;
```

Comment:
cs. is only for old-style config params, for which we should not add any new ones. So no changes to the cs. part should be necessary.